src/pollable_input_stream.rs

```diff
@@ -166,16 +162,15 @@ impl<O: IsA<PollableInputStream>> PollableInputStreamExtManual for O {
     ) -> Pin<Box<dyn Stream<Item = ()> + 'static>> {
         let cancellable: Option<Cancellable> = cancellable.map(|c| c.as_ref()).cloned();
 
-        let obj = Fragile::new(self.clone());
+        let obj = self.clone();
         Box::pin(glib::SourceStream::new(move |send| {
-            obj.get()
-                .create_source(cancellable.as_ref(), None, priority, move |_| {
-                    if send.unbounded_send(()).is_err() {
-                        glib::Continue(false)
-                    } else {
-                        glib::Continue(true)
-                    }
-                })
+            obj.create_source(cancellable.as_ref(), None, priority, move |_| {
+                if send.unbounded_send(()).is_err() {
```

Comment:
Why not `glib::Continue(send.unbounded_send(()).is_ok())`?

Reply:
Harder to read :)